### PR TITLE
refactor code to es6 arrow functions wherever needed

### LINKED ISSUE #182: refactor code to es6 arrow functions
Some of the functions are written in the traditional way, while some are ES6 arrow functions. Changing all functions to arrow functions where they don't cause binding problems could help increase readability

### PR DESCRIPTION
**Pull request by architkshk**
Fixes #182 @abhishek97 Please review.

### CODE REVIEW

File: src/middlewares/acl.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
-function ensureAdmin(req, res, next) {
+ensureAdmin = (req, res, next) => {
```
Comment (championswimmer) [inline src/middlewares/acl.js]: let? const ? nothing ? wtf

File: src/routers/api/users.js [hunk 2]
```diff
@@ -112,18 +112,18 @@ router.get('/me/address',
...
-            }).then(function () {
+            }).then( () => {
                 res.status(202).send({
                     'user_id': req.user.id,
                     'logout': 'success'
                 })
-            }).catch(function (err) {
+            }).catch( (err) => {
                 res.status(501).send(err)
             })
```
Comment (championswimmer) [inline src/routers/api/users.js]: what is the point if you are not taking advantage of arrow functions ? 

```js
}).catch(res.status(501).send)
```

File: src/routers/disconnect/facebook.js [hunk 3]
```diff
@@ -4,7 +4,7 @@
 const router = require('express').Router()
 const models = require('../../db/models').models
 
-function DisconnectFacebook(req, res) {
+DisconnectFacebook = (req, res) => {
```
Comment (championswimmer) [inline src/routers/disconnect/facebook.js]: let? const ? anything ?

File: src/routers/disconnect/github.js [hunk 4]
```diff
@@ -4,7 +4,7 @@
 const router = require('express').Router()
 const models = require('../../db/models').models
 
-function DisconnectGithub(req, res) {
+DisconnectGithub = (req, res) => {
```
Comment (championswimmer) [inline src/routers/disconnect/github.js]: same

File: src/routers/disconnect/google.js [hunk 5]
```diff
@@ -1,7 +1,7 @@
 const router = require('express').Router()
 const models = require('../../db/models').models
 
-function DisconnectGoogle(req, res) {
+DisconnectGoogle  = (req, res) => {
```
Comment (championswimmer) [inline src/routers/disconnect/google.js]: same

File: src/routers/login/facebook.js [hunk 6]
```diff
@@ -7,7 +7,7 @@ const passport = require('../../passport/passporthandler')
 const config = require('../../../config')
 const debug = require('debug')('oauth:login:facebook')
 
-function authnOrAuthzFacebook(req, res, next) {
+authnOrAuthzFacebook  = (req, res, next) => {
```
Comment (championswimmer) [inline src/routers/login/facebook.js]: ???

File: src/routers/login/github.js [hunk 7]
```diff
@@ -7,7 +7,7 @@ const router = require('express').Router()
 const passport = require('../../passport/passporthandler')
 const debug = require('debug')('oauth:login:github')
 
-function authnOrAuthzGithub(req, res, next) {
+authnOrAuthzGithub = (req, res, next) => {
```
Comment (championswimmer) [inline src/routers/login/github.js]: okay all others like this...

File: src/routers/api/users.js (hunk 2 above)
Comment (architkshk) [reply src/routers/api/users.js]: But isn't `res.status(501).send(err)` a statement and not expression (is not return value of function)??

**Comment by architkshk** [comment]
@championswimmer Sorry, I forgot to push my last commit. Now, it looks fine I guess.

File: src/routers/api/users.js (hunk 2 above)
Comment (championswimmer) [reply src/routers/api/users.js]: it returns undefined, so doesn't matter if you turn it into return. catch is last block, so the returned value is not used anywhere anyway

Comment (architkshk) [reply src/routers/api/users.js]: Ok. Done!!

**Comment by architkshk** [comment]
Done! @championswimmer

**Pull request closed without merging**